### ISSUE TITLE
Add Discounted Cumulative Gain (DCG) metric.

### ISSUE THREAD

**Issue by jshin1394** [google/metrax#48]
Using this keras [implementation](https://www.tensorflow.org/ranking/api_docs/python/tfr/keras/metrics/DCGMetric) as basis, add Discounted Cumulative Gain (DCG) metric to Metrax.

**Comment by jshin1394** [comment]
added.
